src/appleseed-max-impl/appleseedrenderer/projectbuilder.cpp

```diff
@@ -141,11 +140,11 @@ namespace
     }
 
     std::string insert_default_material(
-        asr::Assembly&          assembly,
+        asr::Assembly*          assembly,
```

Comment:
Why this change? `assembly` should never be `nullptr` so a ref is better. Revert.